var/spack/repos/builtin/packages/cereal/package.py

```diff
@@ -66,4 +66,4 @@ def cmake_args(self):
     def install(self, spec, prefix):
         with working_dir(self.build_directory):
             install_tree('doc', prefix.doc)
-            install_tree('include', prefix.include)
+        install_tree('include', prefix.include)
```

Comment:
Would `-DJUST_INSTALL_CEREAL=On` work, too?

Reply:
Seems to work